assignment/api/src/resources/assignment.properties

```diff
@@ -1011,11 +1014,11 @@ content_review.errorFromSource = Troubleshooting information: {0}
 
 content_review.error = An unknown error occurred. The originality report for this submission is not available.
 content_review.error.createAssignment=An error with {0} has occurred while creating this assignment. {1} has saved the assignment in draft mode. Please try posting this assignment again later. {2}
-content_review.note=<div><br /><em>NOTE:</em><ul><li>When submitting attachments, students should only use these file types: {0}.</li><li>Students should always save files with the appropriate extension.</li>{1}</ul></div>
+content_review.note=<div><br /><em>NOTE:</em><ul><li>When submitting attachments, students should only use these file types: {0}.</li><li>Students should always save files with the appropriate extension.</li>{1}{2}</ul></div>
+content_review.note.turnitin=<li>Don't use "&" symbol in the assignment title.</li><li>Select a "Student submission" of your preference (Inline only - Attachments only - Inline and Attachments - Single Uploaded File only).</li><li>Inform students that:</li><ol><li>WPS, Apple Pages, .ODS and .gdoc file formats are not accepted.</li><li>File extension should be included. Text after the dot will be considered as file extension.</li><li>Attachments should not exceed 100 MB in size (.txt attachments should not exceed 2 MB).</li><li>Attachments should occupy less than 800 pages.</li><li>The inclusion of vector images may result in problems when having the "originality report", we recommend .jpg format for images.</li>
```

Comment:
These all seem like points that should be handled through validation; not in a long message that no one will read. 

Why do you tell instructors which file types are accepted and then tell them a few lines later that a few other types aren’t? I think the “only accepted” list is sufficient.

What are you trying to say with the “Select a "Student submission" of your preference (Inline only - Attachments only - Inline and Attachments - Single Uploaded File only)” point? Are you trying to say that these are the only allowed submission types? There is already logic on the page to not allow Turnitin (and I assume all Content Review providers) to be enabled when the “Non electronic” submission type is used. This validation should probably also be applied to the new “External Tool (LTI)” and “Video Submission” types, if they are not supported either. 

The “Inform students” points seem unnecessary because that information is communicated to students on the assignment’s page. Also why should it be the instructor’s responsibility to communicate this? At the most, maybe the instructor should be aware of a couple of these points. 

Therefore, I think this whole line is not needed.